code/backend/batch/BatchPushResults.py

```diff
@@ -53,5 +55,8 @@ def do_batch_push_results(msg: func.QueueMessage) -> None:
         )
     )
     # Process the file
-    document_processor.process(source_url=file_sas, processors=processors)
-    blob_client.upsert_blob_metadata(file_name, {"embeddings_added": "true"})
+    if env_helper.AZURE_SEARCH_USE_INTEGRATED_VECTORIZATION:
+        document_processor.process_using_integrated_vectorisation(source_url=file_sas)
+    else:
+        document_processor.process(source_url=file_sas, processors=processors)
```

Comment:
If this is now in this conditional block, should we pull in the piece of code that creates `processors` too - as that calls out to blob storage where it doesnt need to now